pkg/wallet/api.go

```diff
@@ -20,23 +20,14 @@ type Wallet interface {
 // Crypto interface
 type Crypto interface {
 
-	// CreateEncryptionKey create a new public/private encryption keypair.
+	// CreateKeyPairSet create a new public/private encryption and signagure keypairs combo.
 	//
 	// Returns:
 	//
-	// string: verKey
-	//
-	// error: error
-	CreateEncryptionKey() (string, error)
-
-	// CreateSigningKey create a new public/private signing keypair.
-	//
-	// Returns:
-	//
-	// string: verKey
-	//
+	// string: enc public key of the encryption keypair
+	// string: sig public key of the signature keypair
 	// error: error
-	CreateSigningKey() (string, error)
+	CreateKeyPairSet() (string, string, error)
```

Comment:
TODO: refactor. Don't return two keys but rather a KeySet.

I would consider shortening to CreateKeySet.

Reply:
the issue is I don't want to return a keyset that might include private keys... this will get refactored altogether in #596